Cython/Compiler/ParseTreeTransforms.py

```diff
@@ -1175,6 +1175,7 @@ def visit_AttributeNode(self, node):
     def visit_CallNode(self, node):
         self.visit(node.function)
         if not self.parallel_directive:
+            self.visitchildren(node)
```

Comment:
Note that this will visit the `function` attribute twice. Maybe we can move the explicit one behind the condition?